.changeset/lucky-lies-travel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/button': patch
+'@twilio-paste/core': patch
+---
+
+Updates default size for link buttons to 'default' from 'reset'
```

Comment:
Could you please update this to reflect the change we pivoted to?

Reply:
_great_ point, yes